installer/messages.py

```diff
@@ -0,0 +1,250 @@
+    if dest is not None:
+        dest = Path(dest).expanduser().resolve()
+
+    dest_confirmed = False
+
+    while not dest_confirmed:
+        console.line()
+
+        old_dest = dest
+
+        print(f"InvokeAI will be installed at {dest}")
+        dest_confirmed = Confirm.ask(f"Is this correct?", default="y")
+
+
+        if not dest_confirmed:
+
+            # needs more thought into how to handle this nicely
+            # so that the first selected destination continues to shows up as
+            # default until the user is done selecting (potentially multiple times)
+
+            path_completer = PathCompleter(
+                only_directories=True,
+                expanduser=True,
+                get_paths=lambda: [Path(dest).parent],
+                file_filter=lambda n: not n.startswith("."),
+            )
+            print(f"Please select the destination directory for the installation \[{dest}]: ")
+            selected = prompt(
+                "[Tab] to complete ❯ ",
+                complete_in_thread=True,
+                completer=path_completer,
+                complete_style=CompleteStyle.READLINE_LIKE,
+            )
+            if Path(selected).is_absolute():
+                # use the absolute path directly
+                dest = Path(selected)
+            else:
+                # the user entered a relative path - offer to create it as a sibling to the original destination
+                dest = dest.parent / Path(selected)
+            dest = dest.expanduser().resolve()
+
+        if dest.exists():
+            console.line()
+            print(f":exclamation: Directory {dest} already exists.")
+            console.line()
+            dest_confirmed = Confirm.ask(
+                ":question: Are you sure you want to (re)install in this location?", default="y"
+            )
+            if not dest_confirmed:
+                dest = old_dest
+
+    try:
+        dest.mkdir(exist_ok=True, parents=True)
+        return dest
+    except PermissionError as exc:
+        print(
+            f"Failed to create directory {dest} due to insufficient permissions",
+            style=Style(color="red"),
+            highlight=True,
+        )
+    except OSError as exc:
+        console.print_exception(exc)
+
+    if Confirm.ask("Would you like to try again?"):
+        dest_path(init_path)
+    else:
+        console.rule("Goodbye!")
+
+
+def graphical_accelerator():
+    """
+    Prompt the user to select the graphical accelerator in their system
+    This does not validate user's choices (yet), but only offers choices
+    valid for the platform.
+    CUDA is the fallback.
+    We may be able to detect the GPU driver by shelling out to `modprobe` or `lspci`,
+    but this is not yet supported or reliable. Also, some users may have exotic preferences.
+    """
+
+
+    if ARCH == "arm64" and OS != "Darwin":
+        print(f"Only CPU acceleration is available on {ARCH} architecture. Proceeding with that.")
```

Comment:
I haven't been able to confirm this.... there might be pytorch+cuda+aarch64 wheels on conda-forge, and i know there are custom ones available from Nvidia dev program for the Jetson platform, which is arm64. But pypi seems to only return the CPU build as far as I can tell.

Reply:
The 13.1 wheels on pypi are the cuda wheels:
<img width="798" alt="image" src="https://user-images.githubusercontent.com/40327258/215315149-960e330c-5a02-4bf9-a212-85050cdb7475.png">

Where the CPU Version get's installed via a extra index url:
<img width="798" alt="image" src="https://user-images.githubusercontent.com/40327258/215315175-b38a0763-e2e2-4f9f-9ffc-e57a1e2eb96a.png">

Reply:
So I wonder why the aarch64 wheels on PyPI should be the CPU-Version since I cannot find any hints about this limitation in the docs, neither from nvidia

<img width="548" alt="image" src="https://user-images.githubusercontent.com/40327258/215316074-a6a557e5-54b3-48a8-ba83-43dae5111c71.png">

https://pypi.org/project/torch/#files

Reply:
None of the `aarch64` wheels listed above include CUDA support, which i thought was the whole point of the exercise

Reply:
Then I rly wonder why my runpod instance is using 100% GPU, which would not be possible if cuda is not present, or am I wrong there?